src/client/src/database.rs

```diff
@@ -165,6 +165,13 @@ impl Database {
 
         let mut request = tonic::Request::new(request);
         let metadata = request.metadata_mut();
+        Self::put_hints(metadata, hints)?;
+
+        let response = client.handle(request).await?.into_inner();
+        from_grpc_response(response)
+    }
+
+    fn put_hints(metadata: &mut MetadataMap, hints: &[(&str, &str)]) -> Result<()> {
```

Comment:
It's better to use "x-greptime-hints" instead of "x-greptime-hint-{}".

Reply:
https://github.com/GreptimeTeam/greptimedb/blob/6ab0f0cc5c8e3d0b4c64cc7416533f1a94229e00/src/servers/src/hint_headers.rs#L18

Reply:
i'm trying to align with the current behavior, the hint key is 
`let key = AsciiMetadataKey::from_bytes(format!("x-greptime-hint-{}", key).as_bytes())`
are you suggesting i change it to `x-greptime-hints: auto_create_table=true, ttl=7d` here?

Reply:
yes